addons/l10n_it/data/l10n_it_chart_data.xml

```diff
@@ -1,5 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <data noupdate="1">
+        <!-- Sequence e-invoicing -->
+        <record id="sequence_account_it_invoice" model="ir.sequence">
+            <field name="name">E-invoices sequence</field>
```

Comment:
Italian

Reply:
Remove and use  Unique File Code id of invoice into base 36 something like:
a = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
n=10
s=""
while n:
    (n,m) = divmod(n,len(a))
    s = a[m] + s